### ISSUE TITLE
Develop an alternative text-based file format for cell_fracs data

### ISSUE THREAD

**Issue by gonuke** [pyne/pyne#1116]
Some applications may want to use the information derived by `discretize_geom` in applications that don't want to depend on MOAB.  This might require an alternative text-based format for representing this data.

**Comment by nathanb01** [comment]
@gonuke it seems that discretize_geom takes a mesh as it's input which depends on MOAB. Are you saying that the output of discretize_geom would be generated by an application that does depends on MOAB but then converted to a text based form to be used by other applications?

**Comment by gonuke** [comment]
That's right.  There would be an option to generate some kind of text-based mesh format that included the data from discretize geom.

Note, that this might be possible with MOAB's native VTK writer, if a VTK file was of interest, but there may be other generic/canonical formats, especially since it's a Cartesian grid.

**Comment by gonuke** [comment]
@nathanb01 - I was about to assign this to one of my undergrads, unless you are already digging into it...

**Comment by nathanb01** [comment]
@gonuke go ahead and assign it to one of your undergrads. I will work on something else since I haven't started working on it.
